jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -531,6 +532,86 @@ pipeline {
                         }
                     }
                 }
+                stage('build-and-test-linux-x64-deb') {
+                    when {
+                        beforeAgent true
+                        expression{
+                            params.BUILD_PLATFORM.contains('linux')
+                        }
+                    }
+                    agent { label AGENT_X64 }
```

Comment:
I believe the agent would be Ubuntu right? Or does AL2 work here too?

Reply:
agent is AL2, docker is ubuntu2004 😄

Reply:
Also be aware build is happening on centos7 docker, assemble is happening on ubuntu2004 docker.

Reply:
Is that intended? Or should build be happening on ubuntu too?

Reply:
Build should definitely not happening on Ubuntu, it should happen on centos7 as it has the lowest version of glibc.
Then rpm will assemble on rockylinux8 and deb will assemble on ubuntu2004.
This is by design and it is how it should run.

Reply:
Thanks!